src/server/context.ts

```diff
@@ -484,7 +484,7 @@ export class ServerContext {
         methods: {
           default: async (req, ctx) => {
             if (
-              req.headers.get("referrer")?.includes("/_frsh/error_overlay")
+              req.headers.get("referer")?.includes("/_frsh/error_overlay")
             ) {
               throw new Error("fail");
             }
```

Comment:
Ah I think I accidentally committed that whilst adding the error overlay. It can be removed.
```suggestion
```